src/Resolvers/Microsoft.DotNet.NativeWrapper/EnvironmentProvider.cs

```diff
@@ -41,7 +41,8 @@ private IEnumerable<string> SearchPaths
                     searchPaths.AddRange(
                         _getEnvironmentVariable("PATH")
                         .Split(Path.PathSeparator)
-                        .Select(p => p.Trim('"')));
+                        .Select(p => p.Trim('"'))
+                        .Where(p => !string.IsNullOrWhiteSpace(p)));
```

Comment:
I think you can pass `StringSplitOptions.RemoveEmptyEntries` to the `Split` method instead of adding the additional `Where`